sdk/search/azure-search-documents/azure/search/documents/_internal/_index_documents_batch.py

```diff
@@ -47,11 +51,12 @@ def add_upload_actions(self, *documents):
         :param documents: Documents to upload to an Azure search index. May be
          a single list of documents, or documents as individual parameters.
         :type documents: dict or list[dict]
+        :rtype: List[IndexAction]
```

Comment:
There is no description as to what the method returns (other than the type). Does it return the added actions or the total set of actions?

Reply:
It returns newly added actions. I will update the docstring